java/src/jmri/jmrit/display/layoutEditor/LayoutEditorTools.java

```diff
@@ -652,19 +665,19 @@ private boolean getTurnoutInformation(boolean isCrossover) {
                             || (t.getTurnoutType() == LayoutTurnout.RH_XOVER)
                             || (t.getTurnoutType() == LayoutTurnout.LH_XOVER))
                             && (!isCrossover)) {
-                        javax.swing.JOptionPane.showMessageDialog(layoutEditor,
+                        showMessageDialog(layoutEditor,
```

Comment:
Since you are using JOptionPane elsewhere, please make this ```JOptionPane.showMessageDialog(...)``` and remove the static import for showMessageDialog.

Reply:
Thanks. Just a little overzealous on my pruning… ;-)